### PR TITLE
Move to inline svgs to remove 12 assets.ubuntu.com calls

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- replaced all the assets.ubuntu.com images with inline svgs
- how do we test which is faster?  also the chance of testing async images

## QA

- download the branch
- run `yarn watch`
- load the local `index.html`

## Tests

_I did a quick performance test on local files..._

|Local test|Current|SVG|Change|
|----|-------|---|------|
|Size|70.2kb|75.4kb|-5.3kb|
|Loading ms|3.5 ms|4.8 ms|-1.3 ms|
|Scripting ms|7.6 ms|17.6 ms|-10. ms|
|Rendering ms|8.6 ms|11.9 ms|-3.3 ms|
|Painting ms|1.8 ms|3 ms|-1.2 ms|
|Other ms|54.5 ms|49.7 ms|4.8 ms|
|Overall|76 ms|87 ms|-11 ms|

### CODE REVIEW

**Comment by nottrobin** [comment]
Niiiiiiice!


(Unfortunately, conflicts need resolving...)

**Comment by pmahnke** [comment]
@nottrobin fixed, any thoughts on how we could test this better? see if it helps?

@anthonydillon and this is the chart that made me try... it is a test of the jp.u.c site from tokyo

image

**Comment by anthonydillon** [comment]
Looks good. Nice work :+1:

File: src/js/product-details.js [hunk 1]
```diff
@@ -180,42 +161,40 @@ const canonicalProducts = {
...
+    description: 'Enterprise support service to manage and add value to your Ubuntu
```
Comment (nottrobin) [inline src/js/product-details.js]: Looks like you missed a semicolon

File: src/js/product-details.js [hunk 2]
```diff
@@ -180,42 +161,40 @@ const canonicalProducts = {
...
+    description: 'Enterprise support service to manage and add value to your Ubuntu 
     login: 'http://support.canonical.com/',
```
Comment (nottrobin) [inline src/js/product-details.js]: Not you, but this should be `https`.

**Comment by nottrobin** [comment]
I don't think this needs any further testing. It's clearly vastly superior this way.

Just a couple of points inline.

**Comment by nottrobin** [comment]
For some reason the "Landscape", "Livepatch" and "Snapcraft" login logos aren't displaying in Firefox:

image

I found [this article](https://codepen.io/tigt/post/optimizing-svgs-in-data-uris) on how to format SVGs for inclusion in data-URIs, but it looks like you've already followed all its advice - I think. So I don't know what's going wrong.

**Comment by pmahnke** [comment]
@nottrobin 	good catch, they didn't have an explicit width=32 height=32... they do now.

**Comment by nottrobin** [comment]
Thanks. Now LGTM :+1:

**Pull request merged**